src/ImageSharp/PixelFormats/Alpha8.cs

```diff
@@ -108,10 +114,13 @@ public void ToRgb24(ref Rgb24 dest)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void ToRgba32(ref Rgba32 dest)
         {
-            dest.R = 0;
-            dest.G = 0;
-            dest.B = 0;
-            dest.A = this.PackedValue;
+            dest.PackedValue = this.PackedValue;
+        }
+
+        /// <inheritdoc />
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void ToArgb32(ref Argb32 dest) {
+            dest.PackedValue = (uint)this.PackedValue << 24;
```

Comment:
`dest.A = this.PackedValue` to save some shifting.

Reply:
Changed.